website/docs/en/guide/advanced/environments.mdx

```diff
@@ -100,6 +100,21 @@ export default {
 };
 ```
 
+## Specify environment build
+
+By default, Rsbuild will build all environments in the Rsbuild configuration when you execute `rsbuild dev` or `rsbuild build`. You can build only the specified environment via `--environment <name>`.
+
+```bash
+# Enable build for all environments by default
+rsbuild dev
+
+# Only enable web environment build
+rsbuild dev --environment web
+
+# Enable web and ssr environment build
+rsbuild dev --environment web --environment ssr
```

Comment:
Sometimes we use `ssr` instead of `node` as the environment in our document, should we always use `node`? Using two names is quite confusing.